### PR TITLE
make doc gen repro

### PR DESCRIPTION
**Pull request by mr-c**
- [x] Is it mergable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality, is it tested?
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  and `make doc` output. Use `make format` and manual fixing as needed.
- [x] Is it documented in the ChangeLog?
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by mr-c** [comment]
@ctb review and merge please

File: ChangeLog [hunk 1]
```diff
@@ -26,8 +31,7 @@
 
 2014-11-02  Michael R. Crusoe  <mcrusoe@msu.edu>
 
-   * Doxyfile, Makefile: added Doxygen support, coverage &
-	pylint make targets
+   * Doxyfile, Makefile: added Doxygen support, coverage & pylint make targets
```
Comment (ctb) [inline ChangeLog]: ?

**Comment by ctb** [comment]
LGTM; address question, I'll merge.

File: ChangeLog (hunk 1 above)
Comment (mr-c) [inline ChangeLog]: Just cleaning up the ChangeLog. I didn't write this entry.

**Comment by ctb** [comment]
Thanks!

**Pull request merged**